tests/platform_tests/test_reboot.py

```diff
@@ -87,7 +87,8 @@ def reboot_and_check(localhost, dut, interfaces, xcvr_skip_list,
     # Append the last reboot type to the queue
     logging.info("Append the latest reboot type to the queue")
     REBOOT_TYPE_HISTOYR_QUEUE.append(reboot_type)
-
+    # wait for dut to come back before checking critical processes/intfs
+    wait_for_startup(dut, localhost, delay=10, timeout=800)
```

Comment:
Why is this needed?
My understanding is that check_interfaces_and_services() will call  wait_critical_processes(dut) which calls reset_timeout(dut) that will lookup from the inventory file for the cold boot timeout requirement and use that value if defined.  So perhaps we should update per inventory file for the specific HWSKU that is taking longer instead of making changes here?
Each inventory file for the HWSKU involved should have the "plt_reboot_dict" defined for this purpose...
Please re-evaluate if you can solve the extra time required per HWSKU.  Also, please confirm with the vendor for that HWSKU to see why it is taking longer and whether the extra time is legit...

Reply:
this PR has nothing to do with timeout defined in plt_reboot_dict, it is waiting for dut to start up before doing any `dut.shell()` to check critical services etc

Reply:
This will fix the flaky error I described in PR description.
Also it's mostly seen on SUP, so I made timeout to 800sec, but it will not take that long as it's wait_until